api/v1alpha1/etcdcluster_types.go

```diff
@@ -66,6 +70,11 @@ type EtcdCluster struct {
 	Status EtcdClusterStatus `json:"status,omitempty"`
 }
 
+// calculateQuorumSize returns minimum quorum size for current number of replicas
+func (r *EtcdCluster) calculateQuorumSize() int {
+	return int(math.Ceil(float64(*r.Spec.Replicas) / 2.))
```

Comment:
What will be the quorum of etcd cluster of 3 and 4 nodes?
For 3 nodes it will be 2. For 4 nodes - 3.
As I understand current formula calculates 2 for 3 nodes cluster and 2 for 4 nodes cluster.

Reply:
Thank you for noticing! You're absolulely right. Fixed it :)